### ISSUE TITLE
🐛[BUG] V5 ProTable localization problem

### ISSUE THREAD

**Issue by xXAvoraXx** [ant-design/pro-components#774]
### 🐛 bug 描述
There is a problem converting the table to the local language. When I choose a different language from the language menu, the language of the table changes with Chinese.

### 📷 复现步骤
![page](https://user-images.githubusercontent.com/6988989/96495154-1d1c2700-1250-11eb-9a58-8f46e048876e.PNG)

### © 版本信息

- ProComponents 版本: [e.g. 5.0.0]
- umi 版本
- 浏览器环境
- 开发环境 [e.g. windows, chrome]

**Comment by chenshuai2144** [comment]
give a  repo?

**Comment by xXAvoraXx** [comment]
> give a repo?

I dont use any repo.

**Comment by chenshuai2144** [comment]
Without a reproducible repo, it is difficult for me to find the problem for you

**Comment by xXAvoraXx** [comment]
> Without a reproducible repo, it is difficult for me to find the problem for you

It's actually very simple, I added my local language file to the local folder that came with version five.

First of all, I copied the English language in the local folder and renamed it to Turkish.
![locales](https://user-images.githubusercontent.com/6988989/96907597-1be33800-14a4-11eb-95e5-54df2ee55212.PNG)

After the compilation is complete, the language option automatically appeared in this section.
![lang](https://user-images.githubusercontent.com/6988989/96907584-15ed5700-14a4-11eb-8c30-1fbfbf5694ca.png)

But whenever I make a selection, the language setting of the pro table stays in Chinese there.
